index.md

```diff
@@ -6,7 +6,9 @@ layout: default
 
 ### Status of this document
 
-This document is not the real format specification. It's a simple draft to work. (For a simplified diagram of the layout of a Matroska file, see the [Diagram page.](../diagram/index.html)) But since it's quite complete it is used as a reference for the development of libmatroska. An alternate version of the specification can be found [here](/files/matroska.pdf) (PDF doc maintained by Alexander Noé -- may be outdated).
```

Comment:
- I added a reference to CELLAR to moved some sentence order.
- I suggest Alexander Noé is `outdated` rather than `may be outdated`.